vflow/helpers.py

```diff
@@ -37,21 +37,23 @@ def build_vset(name: str, func, param_dict=None, reps: int = 1,
                is_async: bool = False, output_matching: bool = False,
                lazy: bool = False, cache_dir: str = None, verbose: bool = True,
                tracking_dir: str = None, **kwargs) -> Vset:
-    """Builds a new Vset by currying or instantiating callable `func` with all
```

Comment:
The output should still always be a single `Vset`, regardless of whether `func` is a list or not. Sorry, I know I got this confused during our meeting last week. A typical use case to keep in mind is when you have different learners for the same task (e.g. a few different scikit-learn modeling classes) and also want to do a hyperparameter search for each one or some of them.

Say you have 3 models, but only the first two need a hyperparam sweep. Then the user should pass `param_dict = [param_dict1, None, param_dict3]`. Let's be sure to document this.